src/app.js

```diff
@@ -1,25 +1,12 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+    <React.Fragment>
```

Comment:
why are you using react fragment is unnecessary, addition of extra component can lead to bulky code base

Reply:
They are a necessity to control the Dom tree of the components

Reply:
brother we don't need fragment in this code let all write simply code